flareon/src/auth/db/migrations/m_0001_initial.rs

```diff
@@ -3,7 +3,7 @@
 #[derive(Debug, Copy, Clone)]
 pub(super) struct Migration;
 impl ::flareon::db::migrations::Migration for Migration {
-    const APP_NAME: &'static str = "flareon_auth";
```

Comment:
Out of curiosity, why?

Reply:
Oof, for a moment during the development I thought we need to basically say that each app must have a corresponding crate with the same name (because of reasons which turned out to be useless). I'll revert the change.